bot/irc/mozdefbot.py

```diff
@@ -303,6 +303,8 @@ def on_message(self, body, message):
                     "alertworker exception: unknown body type received %r" % body)
                 return
 
+            bodyDict = full_body['_source']
```

Comment:
Nit: `bodyDict` (here) is called `body_dict` in `bot/slack/mozdefbot.py`.

Reply:
seconded, probably doesn't change the functionality, but consistency is nice.

Reply:
Fixed in https://github.com/mozilla/MozDef/pull/1528/commits/85fd60c3929d62d4a5faa43a4d7841313b1aba38